src/main/scala/it/unibo/authsim/library/user/builder/UserBuilder.scala

```diff
@@ -0,0 +1,34 @@
+package it.unibo.authsim.library.user.builder
+
+import  it.unibo.authsim.library.user.model.User
+import it.unibo.authsim.library.dsl.policy.checkers.StringPolicyChecker
+import it.unibo.authsim.library.dsl.policy.model.StringPolicies.{CredentialPolicy, PasswordPolicy, UserIDPolicy}
+import it.unibo.authsim.library.user.builder.util.Util.generateRandomString
+//ANche qui non sono troppo sicura sull'iso
+abstract class UserBuilder[U]:
+  protected var _credentialPolicies: Seq[CredentialPolicy] = Seq.empty
+  protected var _userName: String=generateRandomString()
+  protected var _password:String=generateRandomString()
```

Comment:
Same as in the other comments: try to avoid the underscore, especially at the beginning of an identifier.